pulseaudio-control.bash

```diff
@@ -26,6 +26,8 @@ SINK_BLACKLIST=(
 declare -A SINK_NICKNAMES
```

Comment:
Now that I think of this, these comments at the top of the file shouldn't explain the options as they aren't meant to be edited now. This should simply initialize the variables with the default values if anything.